src/gui/dialogs/export_project.ui

```diff
@@ -10,6 +10,12 @@
     <height>412</height>
    </rect>
   </property>
+  <property name="minimumSize">
+   <size>
+    <width>519</width>
```

Comment:
To me it would be more odd (statistical anomaly) that a round number were coincidentally the right number.